src/test/osd/osd-scrub-repair.sh

```diff
@@ -925,7 +927,7 @@ function TEST_corrupt_scrub_replicated() {
 EOF
 
     jq "$jqfilter" $dir/json | python -c "$sortkeys" | sed -e "$sedfilter" > $dir/csjson
-    diff -y $termwidth $dir/checkcsjson $dir/csjson || test $getjson = "yes" || return 1
+    diff ${DIFFCOLOPTS} $dir/checkcsjson $dir/csjson || test $getjson = "yes" || return 1
```

Comment:
@wjwithagen, oh, i didn't mean to use `diff` here. it would cause stack overflow if bash fails to detect this.